vue/components/ui/molecules/avatar-list/avatar-list.vue

```diff
@@ -0,0 +1,166 @@
+<template>
+    <div class="avatar-list">
+        <div
+            class="avatar-container"
+            v-bind:class="classes"
+            v-bind:style="{ 'z-index': index }"
+            v-for="(avatar, index) in avatarList"
+            v-bind:key="index"
+        >
+            <div v-if="index == maxElems">
+                {{ remainingElements }}
+            </div>
+            <avatar v-bind="{ size: size, ...avatar }" v-else-if="index < maxElems" />
+        </div>
+    </div>
+</template>
+
+<style lang="scss" scoped>
+.avatar-list {
+    display: flex;
+}
+
+.avatar-list > .avatar-container > * {
+    border: 2px solid #ffffff;
+}
+
+.avatar-list > .avatar-container:not(:first-child) {
+    transition: margin 0.3s;
+}
+
+.avatar-list > .avatar-container:not(:first-child).very-tiny {
+    margin-left: -10px;
+}
+
+.avatar-list > .avatar-container:not(:first-child).tiny {
+    margin-left: -12px;
+}
+
+.avatar-list > .avatar-container:not(:first-child).small {
+    margin-left: -16px;
+}
+
+.avatar-list > .avatar-container:not(:first-child).medium {
+    margin-left: -18px;
+}
+
+.avatar-list > .avatar-container:not(:first-child).large {
+    margin-left: -26px;
+}
+
+.avatar-list > .avatar-container:not(:first-child).very-large {
+    margin-left: -50px;
+}
+
+.avatar-list:hover > .avatar-container.expandable:not(:first-child).very-tiny {
+    margin-left: 2px;
+}
+
+.avatar-list:hover > .avatar-container.expandable:not(:first-child).tiny {
+    margin-left: 2px;
+}
+
+.avatar-list:hover > .avatar-container.expandable:not(:first-child).small {
+    margin-left: 4px;
+}
+
+.avatar-list:hover > .avatar-container.expandable:not(:first-child).medium {
+    margin-left: 6px;
+}
+
+.avatar-list:hover > .avatar-container.expandable:not(:first-child).large {
+    margin-left: 8px;
+}
+
+.avatar-list:hover > .avatar-container.expandable:not(:first-child).very-large {
+    margin-left: 10px;
+}
+
+.avatar-list > .avatar-container > div {
+    align-content: center;
+    background: #e1e1e1;
+    border-radius: 50%;
+    color: #545454;
+    display: flex;
+    justify-content: center;
+}
+
+.avatar-list > .avatar-container.very-tiny > div {

```

Comment:
Make the selector more specific, avoid using general element tags. It applies to the rest of selectors that have the ` > div`